aptos-move/aptos-aggregator/src/resolver.rs

```diff
@@ -18,29 +28,89 @@ pub enum AggregatorReadMode {
 /// Returns a value of an aggregator from cache or global storage.
 ///   - Ok(..)       if aggregator value exists
 ///   - Err(..)      otherwise.
-pub trait AggregatorResolver {
+pub trait TAggregatorView {
+    type Identifier;
```

Comment:
So if we separate V1 and V2 querying interfaces, might make sense to have V1 and V2 identifier types? but having the AggregatorID that is an enum and extracting the correct one should also be fine. And both are easy to adjust to when we deprecate V1.